.buildkite/nightly-benchmarks/serving-tests.json

```diff
@@ -0,0 +1,70 @@
+[
+    {
+        "test_name": "serving_llama8B_tp1_sharegpt",
+        "server_parameters": {
+            "model": "meta-llama/Meta-Llama-3-8B",
+            "tensor_parallel_size": 1,
+            "swap_space": 16,
+            "disable_log_stats": "",
+            "disable_log_requests": "",
+            "load_format": "dummy"
+        },
+        "client_parameters": {
+            "model": "meta-llama/Meta-Llama-3-8B",
+            "dataset_name": "sharegpt",
+            "dataset_path": "./ShareGPT_V3_unfiltered_cleaned_split.json",
+            "num_prompts": 1000
+        }
+    },
+    {
+        "test_name": "serving_llama70B_tp4_sharegpt",
+        "server_parameters": {
+            "model": "meta-llama/Meta-Llama-3-70B-Instruct",
+            "tensor_parallel_size": 4,
+            "swap_space": 16,
+            "disable_log_stats": "",
+            "disable_log_requests": "",
+            "load_format": "dummy"
+        },
+        "client_parameters": {
+            "model": "meta-llama/Meta-Llama-3-70B-Instruct",
+            "dataset_name": "sharegpt",
+            "dataset_path": "./ShareGPT_V3_unfiltered_cleaned_split.json",
+            "num_prompts": 1000
+        }
+    },
+    {
+        "test_name": "serving_mixtral8x7B_tp2_sharegpt",
+        "server_parameters": {
+            "model": "mistralai/Mixtral-8x7B-Instruct-v0.1",
+            "tensor_parallel_size": 2,
```

Comment:
Why tp 2? (unlike llama8b?)

Reply:
Mixtral is a mixture-of-expert model so the # of params is actually 8x7B not 7B. Plus, the main goal of this test is to test the performance of custom mixture-of-expert kernel under distributed settings. So I set tp=2. I also heard that tp=2 is a common setup for mixtral 8x7B model.

Reply:
oh I see. We were using tp 4 internally, so I was just curious. tp 2 sounds good to me too.